bin/api-docs/gen-theme-reference.js

```diff
@@ -127,16 +127,44 @@ const getSettingsPropertiesMarkup = ( struct ) => {
 	}
 
 	let markup = '| Property  | Type   | Default | Props  |\n';
-	markup += '| ---       | ---    | ---    |---   |\n';
+	markup += '| ---    | ---    | ---    |---   |\n';
```

Comment:
I also tried to add the description in here to see what it would look like. I think the description would help in this documentation, but it was very wordy.

Reply:
I agree 👍